src/components/Trade/Components/TradeQuotes/TradeQuote.tsx

```diff
@@ -161,6 +159,27 @@ export const TradeQuoteLoaded: React.FC<TradeQuoteLoadedProps> = ({
     return borderColor
   })()
 
+  const protocolIcon = useMemo(() => {
+    switch (swapperWithMetadata.swapper.getType()) {
```

Comment:
Actually agree with @0xApotheosis not to call this inline within the switch, even though this shouldn't throw, this is JS so always better be safe